### PR TITLE
Sos 136 unify the buttons

### PR DESCRIPTION
**Pull request by SeanTConrad**
Cleaned up some buttons and cards. Fixed bug on Employees page.

### CODE REVIEW

File: client/src/components/AccountCard.tsx [hunk 1]
```diff
@@ -1,75 +1,81 @@
...
-  const { id, primary_color } = props.account;
-  const { deleteAccountById, unDeleteAccountById } = useAccounts();
+    const toggleShowTransactions = () => {
+        setTransactionsShown(shown => !shown);
+    };
 
-  const toggleShowTransactions = () => {
-    setTransactionsShown(shown => !shown);
-  };
+    useEffect(() => {
```
Comment (aylusltd) [inline client/src/components/AccountCard.tsx:29]: Doesn't this cause a loop? getTransactionsByAccount should update transactionsByAccount.

File: client/src/components/EmployeesPage.tsx [hunk 2]
```diff
@@ -72,11 +72,21 @@ export default function EmployeesPage(){
...
 
+    // Get Employee info
+    useEffect(()=>{
+        if(companyId)
+            getEmployeesByCompany(companyId)
+    },[getEmployeesByCompany])
+    useEffect(() => {
+        if(employeesByCompany[companyId])
+            setEmployees(Object.values(employeesByCompany[companyId]));
+    }, [employeesByCompany, getEmployeesByCompany]);
```
Comment (aylusltd) [inline client/src/components/EmployeesPage.tsx:83]: this should also cause a loop

File: client/src/services/api.tsx [hunk 3]
```diff
@@ -65,7 +65,7 @@ export const getCurrentUser = () => {
 };
 
 const simulationUser : UserType = {
-    id: 6,
+    id: 1,
```
Comment (aylusltd) [inline client/src/services/api.tsx:68]: Is this on purpose? I'd rather you not simulate me as the user ID... It's hard enough to debug already.

**Review by aylusltd — CHANGES_REQUESTED** [review]

File: client/src/services/api.tsx (hunk 3 above)
Comment (SeanTConrad) [reply client/src/services/api.tsx:68]: Sorry. I can set this back. I was trying to QA the Employees page.

File: client/src/components/AccountCard.tsx (hunk 1 above)
Comment (SeanTConrad) [reply client/src/components/AccountCard.tsx:29]: I only updated the tabs on this code. This is the useEffect running in the current Beta branch, unless I'm misreading the full diff.  However, if should cause a loop. It doesn't though. I removed it and it still all works, so keeping it removed.

I'll isolate tab updates and some console.log removal to its own branch so it's easier to approve.

File: client/src/components/EmployeesPage.tsx (hunk 2 above)
Comment (SeanTConrad) [reply client/src/components/EmployeesPage.tsx:83]: Removed and tested. Still works.  This was also the existing way it was.

File: client/src/components/AccountCard.tsx (hunk 1 above)
Comment (SeanTConrad) [reply client/src/components/AccountCard.tsx:29]: Fix committed

File: client/src/components/EmployeesPage.tsx (hunk 2 above)
Comment (SeanTConrad) [reply client/src/components/EmployeesPage.tsx]: Fix commited

**Review by aylusltd — APPROVED** [review]

**Pull request merged**